pkgs/development/haskell-modules/configuration-ghc-9.2.x.nix

```diff
@@ -218,9 +222,7 @@ self: super: {
 
   # hlint 3.3 needs a ghc-lib-parser newer than the one from stackage
   hlint = super.hlint_3_3_4.overrideScope (self: super: {
-    ghc-lib-parser = overrideCabal self.ghc-lib-parser_9_0_1_20210324 {
-      doHaddock = false;
-    };
-    ghc-lib-parser-ex = self.ghc-lib-parser-ex_9_0_0_4;
+    ghc-lib-parser = self.ghc-lib-parser_9_2_1_20211101;
+    ghc-lib-parser-ex = self.ghc-lib-parser-ex_9_2_0_1;
```

Comment:
From my testing this doesn't compile this way, have you actually tested this change?